From 1fa6b1706cb0cf70206da5687e79a4c28737bec8 Mon Sep 17 00:00:00 2001
From: Shengjiu Wang <shengjiu.wang@freescale.com>
Date: Tue, 6 Jun 2017 11:28:18 +0800
Subject: [PATCH 1863/5242] MLK-15011: ARM64: dts: increase the cma size for
 imx8qxp

commit  32ed17cad75c9663b5133ff80cafc46932106730 from
https://source.codeaurora.org/external/imx/linux-imx.git

Current the CMA size is 128M, after GPU enabled, there is no
space left for other drivers. So increase it to 640M, which is
align with imx8qm

Signed-off-by: Shengjiu Wang <shengjiu.wang@freescale.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8qxp.dtsi |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qxp.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8qxp.dtsi
index 3167b9a..b1c7bdd 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qxp.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qxp.dtsi
@@ -52,7 +52,7 @@
 		linux,cma {
 			compatible = "shared-dma-pool";
 			reusable;
-			size = <0 0x08000000>;
+			size = <0 0x28000000>;
 			alloc-ranges = <0 0x80000000 0 0x80000000>;
 			linux,cma-default;
 		};
-- 
1.7.9.5

